caas/containers.go

```diff
@@ -36,5 +45,13 @@ func ParseContainerSpec(in string) (*ContainerSpec, error) {
 	if spec.ImageName == "" {
 		return nil, errors.New("spec image name is missing")
 	}
+	for _, fs := range spec.Files {
+		if fs.Name == "" {
+			return nil, errors.New("file spec name is missing")
+		}
+		if fs.MountPath == "" {
+			return nil, errors.Errorf("mount path is missing for file spec %q", fs.Name)
+		}
+	}
```

Comment:
Why not check to see if files are provided here?

Reply:
files might be added later; as a minimum we just set up the mount